src/db/mongo.ts

```diff
@@ -0,0 +1,10 @@
+import mongoose from 'mongoose'
+import { mongoConfig } from '../config'
+
+export const createDBConnection = () => {
```

Comment:
export const createDBConnection = () => mongoose.connect(`${mongoConfig.mongoUrl}`, {
        useNewUrlParser: true,
        useCreateIndex: true,
        useUnifiedTopology: true
    })
Change like this

Reply:
Okay. Doing htat.

Reply:
Can we also rename it to createMongoConnection

Reply:
Okay sure.